src/import/sst_service.rs

```diff
@@ -368,14 +353,13 @@ where
     }
 
     fn ingest_files(
-        &self,
-        context: Context,
+        &mut self,
+        mut context: Context,
         label: &'static str,
         ssts: Vec<SstMeta>,
     ) -> impl Future<Output = Result<IngestResponse>> {
-        let header = make_request_header(context);
-        let snapshot_res = Self::async_snapshot(self.router.clone(), header.clone());
-        let router = self.router.clone();
+        let snapshot_res = Self::async_snapshot(&mut self.engine, &context);
```

Comment:
> // Here we shall check whether the file has been ingested before. This operation
            // must execute after geting a snapshot from raftstore to make sure that the
            // current leader has applied to current term.

Seems the snapshot is to make sure applied relation only for the ingested region. When the sst_service is implemented for server2, can we use local_backend engine to get a snapshot?

Reply:
Local backend doesn't provide such guarantee. Querying `Engine` is the expected usage.
> to make sure that the current leader has applied to current term.